web/models.py

```
@@ -2921,3 +2921,51 @@ class Discount(models.Model):
 
     def __str__(self):
         return f"{self.code} for {self.user.username} on {self.course.title}"
+
+
+class Survey(models.Model):
+    title = models.CharField(max_length=200)
+    author = models.ForeignKey(User, on_delete=models.SET_NULL, blank=True, null=True)  # Added null=True
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)